pro/src/new_components/OfferIndividualForm/UsefulInformations/Venue/__specs__/Venue.spec.tsx

```diff
@@ -0,0 +1,254 @@
+        name: 'Offerer AA',
+      }) as HTMLOptionElement
+      expect(offererDefaultOption.selected).toBeTruthy()
+      expect(selectVenue).toBeDisabled()
+      expect(selectOfferer.options.length).toBe(1)
+
+      const venueDefaultOption = screen.getByRole('option', {
+        name: 'Venue AAAA',
+      }) as HTMLOptionElement
+      expect(venueDefaultOption.selected).toBeTruthy()
+      expect(selectVenue).toBeDisabled()
+      expect(selectVenue.options.length).toBe(1)
+    })
+  })
+
+  it('test select a offerer id', async () => {
+    const { selectVenue, selectOfferer } = renderVenue({
+      initialValues,
+      onSubmit,
+      props,
+    })
+
+    let selectedOffererOption
+    let selectedVenueOption
+
+    // select a offerer with 1 venue
+    await userEvent.selectOptions(selectOfferer, 'AA')
+    selectedOffererOption = screen.getByRole('option', {
+      name: 'Offerer AA',
+    }) as HTMLOptionElement
+    expect(selectedOffererOption.selected).toBeTruthy()
+
+    expect(selectVenue).toBeDisabled()
+    expect(selectVenue.options.length).toBe(1)
+    selectedVenueOption = screen.getByRole('option', {
+      name: 'Venue AAAA',
+    }) as HTMLOptionElement
+    expect(selectedVenueOption.selected).toBeTruthy()
+
+    // select a other offerer with 1 venue
+    await userEvent.selectOptions(selectOfferer, 'BB')
+    selectedOffererOption = screen.getByRole('option', {
+      name: 'Offerer BB',
+    }) as HTMLOptionElement
+    expect(selectedOffererOption.selected).toBeTruthy()
+
+    expect(selectVenue).toBeDisabled()
+    expect(selectVenue.options.length).toBe(1)
+    selectedVenueOption = screen.getByRole('option', {
+      name: 'Venue BBAA',
+    }) as HTMLOptionElement
+    expect(selectedVenueOption.selected).toBeTruthy()
+
+    // select a other offerer with 2 venue
+    await userEvent.selectOptions(selectOfferer, 'CC')
+    selectedOffererOption = screen.getByRole('option', {
+      name: 'Offerer CC',
+    }) as HTMLOptionElement
+    expect(selectedOffererOption.selected).toBeTruthy()
+
+    expect(selectVenue).not.toBeDisabled()
+    expect(selectVenue.options.length).toBe(3)
+    selectedVenueOption = screen.getByRole('option', {
+      name: 'Selectionner un lieu',
+    }) as HTMLOptionElement
+    expect(selectedVenueOption.selected).toBeTruthy()
+
+    // unselect offerer id
+    await userEvent.selectOptions(selectOfferer, 'Selectionner une structure')
+    selectedOffererOption = screen.getByRole('option', {
+      name: 'Selectionner une structure',
+    }) as HTMLOptionElement
+    expect(selectedOffererOption.selected).toBeTruthy()
+
+    expect(selectVenue).toBeInTheDocument()
+    const venueDefaultOption = screen.getByRole('option', {
+      name: 'Selectionner un lieu',
+    }) as HTMLOptionElement
+    expect(venueDefaultOption.selected).toBeTruthy()
+    expect(selectVenue).toBeDisabled()
+    expect(selectVenue.options.length).toBe(1)
+  })
+
+  it('test required fields errors', async () => {
```

Comment:
`should display an error when required fields are not filled` ? au bon coin on recommendait d'écrire les intitulés de tests orientés utilisateur, cela permet de décrire plus précisément ce qu'on teste

Reply:
modification faite [ici](https://github.com/pass-culture/pass-culture-main/pull/2227/commits/06c9da4b6c9e7d662affb2c3c5a89bebc6febf6a)